scipy/stats/_multivariate.py

```diff
@@ -5075,3 +5076,157 @@ def rvs(self, size=1, random_state=None):
+    r"""A vector-valued random direction.
+
+    Return a random direction (unit vector).
+
+    The `dim` keyword specifies the dimension.
+
+    Methods
+    -------
+    rvs(dim=None, size=1, random_state=None)
+        Draw random directions.
+
+    Parameters
+    ----------
+    dim : scalar
+        Dimension of directions.
+    seed : {None, int, `numpy.random.Generator`,
+            `numpy.random.RandomState`}, optional
+
+        Used for drawing random variates.
+        If `seed` is `None`, the `~np.random.RandomState` singleton is used.
+        If `seed` is an int, a new ``RandomState`` instance is used, seeded
+        with seed.
+        If `seed` is already a ``RandomState`` or ``Generator`` instance,
+        then that object is used.
+        Default is `None`.
+
+    Notes
+    -----
+    This distribution generates unit vectors uniformly distributed on
+    the surface of a sphere. These can be interpreted as random directions.
+
+    For example, if `dim` is 3, 3D vectors from the surface of :math:`S^2`
+    will be sampled.
+
+    References
+    ----------
+    .. [1] Marsaglia, G. (1972). "Choosing a Point from the Surface of a
+           Sphere". Annals of Mathematical Statistics. 43 (2): 645-646.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from scipy.stats import random_direction
+    >>> x = random_direction.rvs(3)
+    >>> np.linalg.norm(x)
+    1.
+
+    This generates one random direction, a vector on the surface of
+    :math:`S^2`.
+
+    Alternatively, the object may be called (as a function) to fix the `dim`
+    parameter, return a "frozen" `random_direction` random variable:
+
+    >>> rv = random_direction(5)
+
+    """
+
+    def __init__(self, seed=None):
+        super().__init__(seed)
+        self.__doc__ = doccer.docformat(self.__doc__)
+
+    def __call__(self, dim=None, seed=None):
+        """Create a frozen n-dimensional uniform direction distribution.
+
+        See `random_direction` for more information.
+        """
+        return random_direction_frozen(dim, seed=seed)
+
+    def _process_parameters(self, dim):
+        """Dimension N must be specified; it cannot be inferred."""
+        if dim is None or not np.isscalar(dim) or dim < 1 or dim != int(dim):
+            raise ValueError("Dimension of vector must be specified, "
+                             "and must be an integer greater than 0.")
+
+        return dim
+
+    def rvs(self, dim, size=1, random_state=None):
+        """Draw random samples from S(N-1).
+
+        Parameters
+        ----------
+        dim : integer
+            Dimension of space (N).
+        size : int or tuple of ints, optional
+            Given a shape of, for example, (m,n,k), m*n*k samples are
+            generated, and packed in an m-by-n-by-k arrangement.
+            Because each sample is N-dimensional, the output shape
+            is (m,n,k,N). If no shape is specified, a single (N-D)
+            sample is returned.
+
```

Comment:
`random_state` parameter should be documented.

Reply:
I did not see that parameter documented for the other distributions. Below For example `multivariate_normal`. I am not sure it would be properly rendered actually.

Still, documenting `random_state` would totally make sense Not sure how stats' documentary machinery works but I think it would make sense to automatically add that to all `.rvs` methods if possible. 

![image](https://user-images.githubusercontent.com/40656107/198822574-c6bd3404-9f4b-4d52-bd59-a31879f8a95b.png)

Reply:
It doesn't get rendered online, but I think for some distributions it shows up in IDEs and with `help`. 
In any case, I'd suggest just adding it manually. There are other problems with multivariate documentation machinery (especially a confusion of `seed` and `random_state`) that I'd rather not get into here.

Reply:
Ok, now I am also confused. What is the difference between `seed` and `random_state` then? Never had to bother with that, I guess like 99% of people I just set an integer seed if I ever needed :D.

Is it documented for another distribution somerwhere maybe? Then I would gladly add it here as well.

Reply:
> ...that I'd rather not get into here.

But if you insist : )

Execute:
```python3
from scipy.stats import multivariate_normal
help(multivariate_normal.rvs)
```
You get:
<details>

```
Help on method rvs in module scipy.stats._multivariate:

rvs(mean=None, cov=1, size=1, random_state=None) method of scipy.stats._multivariate.multivariate_normal_gen instance
    Draw random samples from a multivariate normal distribution.
    
    Parameters
    ----------
    mean : array_like, default: ``[0]``
        Mean of the distribution.
    cov : array_like or `Covariance`, default: ``[1]``
        Symmetric positive (semi)definite covariance matrix of the distribution.
    allow_singular : bool, default: ``False``
        Whether to allow a singular covariance matrix. This is ignored if `cov` is
        a `Covariance` object.
    size : integer, optional
        Number of samples to draw (default 1).
    seed : {None, int, np.random.RandomState, np.random.Generator}, optional
        Used for drawing random variates.
        If `seed` is `None`, the `~np.random.RandomState` singleton is used.
        If `seed` is an int, a new ``RandomState`` instance is used, seeded
        with seed.
        If `seed` is already a ``RandomState`` or ``Generator`` instance,
        then that object is used.
        Default is `None`.
    
    Returns
    -------
    rvs : ndarray or scalar
        Random variates of size (`size`, `N`), where `N` is the
        dimension of the random variable.
    
    Notes
    -----
    Setting the parameter `mean` to `None` is equivalent to having `mean`
    be the zero-vector. The parameter `cov` can be a scalar, in which case
    the covariance matrix is the identity times that value, a vector of
    diagonal entries for the covariance matrix, a two-dimensional array_like,
    or a `Covariance` object.
```

</details>

The `rvs` method mistakenly documents a `seed` parameter that is not accepted by the method. That description is intended to document the `random_state` parameter. If you substitute `random_state` for `seed` in the description, it's correct.

`seed` _is_ the name of the equivalent parameter of the `__call__` method that is used to freeze the distribution. That is documented "correctly" for most distributions (as a "parameter" of the object).

<details>

```python3
help(multivariate_normal)
```

```
Help on multivariate_normal_gen in module scipy.stats._multivariate:

<scipy.stats._multivariate.multivariate_normal_gen object>
    A multivariate normal random variable.
    
    The `mean` keyword specifies the mean. The `cov` keyword specifies the
    covariance matrix.
    
    Methods
    -------
    pdf(x, mean=None, cov=1, allow_singular=False)
        Probability density function.
    logpdf(x, mean=None, cov=1, allow_singular=False)
        Log of the probability density function.
    cdf(x, mean=None, cov=1, allow_singular=False, maxpts=1000000*dim, abseps=1e-5, releps=1e-5, lower_limit=None)  # noqa
        Cumulative distribution function.
    logcdf(x, mean=None, cov=1, allow_singular=False, maxpts=1000000*dim, abseps=1e-5, releps=1e-5)
        Log of the cumulative distribution function.
    rvs(mean=None, cov=1, size=1, random_state=None)
        Draw random samples from a multivariate normal distribution.
    entropy()
        Compute the differential entropy of the multivariate normal.
    
    Parameters
    ----------
    mean : array_like, default: ``[0]``
        Mean of the distribution.
    cov : array_like or `Covariance`, default: ``[1]``
        Symmetric positive (semi)definite covariance matrix of the distribution.
    allow_singular : bool, default: ``False``
        Whether to allow a singular covariance matrix. This is ignored if `cov` is
        a `Covariance` object.
    seed : {None, int, np.random.RandomState, np.random.Generator}, optional
        Used for drawing random variates.
        If `seed` is `None`, the `~np.random.RandomState` singleton is used.
        If `seed` is an int, a new ``RandomState`` instance is used, seeded
        with seed.
        If `seed` is already a ``RandomState`` or ``Generator`` instance,
        then that object is used.
        Default is `None`.
    
    Notes
    -----
    Setting the parameter `mean` to `None` is equivalent to having `mean`
    be the zero-vector. The parameter `cov` can be a scalar, in which case
    the covariance matrix is the identity times that value, a vector of
    diagonal entries for the covariance matrix, a two-dimensional array_like,
    or a `Covariance` object.
    
    The covariance matrix `cov` may be an instance of a subclass of
    `Covariance`, e.g. `scipy.stats.CovViaPrecision`. If so, `allow_singular`
    is ignored.
    
    Otherwise, `cov` must be a symmetric positive semidefinite
    matrix when `allow_singular` is True; it must be (strictly) positive
    definite when `allow_singular` is False.
    Symmetry is not checked; only the lower triangular portion is used.
    The determinant and inverse of `cov` are computed
    as the pseudo-determinant and pseudo-inverse, respectively, so
    that `cov` does not need to have full rank.
    
    The probability density function for `multivariate_normal` is
    
    .. math::
    
        f(x) = \frac{1}{\sqrt{(2 \pi)^k \det \Sigma}}
               \exp\left( -\frac{1}{2} (x - \mu)^T \Sigma^{-1} (x - \mu) \right),
    
    where :math:`\mu` is the mean, :math:`\Sigma` the covariance matrix,
    :math:`k` the rank of :math:`\Sigma`. In case of singular :math:`\Sigma`,
    SciPy extends this definition according to [1]_.
    
    .. versionadded:: 0.14.0
    
    References
    ----------
    .. [1] Multivariate Normal Distribution - Degenerate Case, Wikipedia,
           https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Degenerate_case
    
    Examples
    --------
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import multivariate_normal
    
x = np.linspace(0, 5, 10, endpoint=False)
y = multivariate_normal.pdf(x, mean=2.5, cov=0.5); y
    array([ 0.00108914,  0.01033349,  0.05946514,  0.20755375,  0.43939129,
            0.56418958,  0.43939129,  0.20755375,  0.05946514,  0.01033349])
fig1 = plt.figure()
ax = fig1.add_subplot(111)
ax.plot(x, y)
plt.show()
    
    Alternatively, the object may be called (as a function) to fix the mean
    and covariance parameters, returning a "frozen" multivariate normal
    random variable:
    
rv = multivariate_normal(mean=None, cov=1, allow_singular=False)
# Frozen object with the same methods but holding the given
# mean and covariance fixed.
    
    The input quantiles can be any shape of array, as long as the last
    axis labels the components.  This allows us for instance to
    display the frozen pdf for a non-isotropic random variable in 2D as
    follows:
    
x, y = np.mgrid[-1:1:.01, -1:1:.01]
pos = np.dstack((x, y))
rv = multivariate_normal([0.5, -0.2], [[2.0, 0.3], [0.3, 0.5]])
fig2 = plt.figure()
ax2 = fig2.add_subplot(111)
ax2.contourf(x, y, rv.pdf(pos))
```

</details>

I haven't looked into the history of how they got named differently, but that's the state of things. I didn't really want to mention it because I don't want this PR to get held up trying to fix it for all distributions. So the suggestion was that you manually copy-paste a correct description of `random_state` into the docstring of this distribution's `rvs` method rather than trying to use the docstring magic seen throughout the file.